aspnetcore/blazor/index.md

```diff
@@ -57,102 +57,21 @@ Using .NET for client-side web development offers the following advantages:
-
-    [Parameter]
-    public string? Title { get; set; }
-
-    private void OnYes()
-    {
-        Console.WriteLine("Write to the console in C#! 'Yes' button selected.");
-    }
-}
-```
-
-In the preceding example, `OnYes` is a C# method triggered by the button's `onclick` event. The dialog's text (`ChildContent`) and title (`Title`) are provided by the following component that uses this component in its UI.
-
-The `Dialog` component is nested within another component using an HTML tag. In the following example, the `Index` component (`Index.razor`) uses the preceding `Dialog` component. The tag's `Title` attribute passes a value for the title to the `Dialog` component's `Title` property.  The `Dialog` component's text (`ChildContent`) are set by the content of the `<Dialog>` element. When the `Dialog` component is added to the `Index` component, [IntelliSense in Visual Studio](/visualstudio/ide/using-intellisense) speeds development with syntax and parameter completion.
+<!-- Start >=8.0 coverage -->
 
 :::moniker range=">= aspnetcore-8.0"
 
-```razor
-@page "/"
-@attribute [RenderModeServer]
-
-<h1>Hello, world!</h1>
-
-<p>
-    Welcome to your new app.
-</p>
-
-<Dialog Title="Learn More">
-    Do you want to <i>learn more</i> about Blazor?
-</Dialog>
-```
-
-:::moniker-end
-
-:::moniker range="< aspnetcore-8.0"
-
-```razor
-@page "/"
-
-<h1>Hello, world!</h1>
-
-<p>
-    Welcome to your new app.
-</p>
-
-<Dialog Title="Learn More">
-    Do you want to <i>learn more</i> about Blazor?
-</Dialog>
-```
-
-:::moniker-end
-
-The dialog is rendered when the `Index` component is accessed in a browser. When the button is selected by the user, the browser's developer tools console shows the message written by the `OnYes` method:
-
-![Dialog component rendered in the browser nested inside of the Index component. The browser developer tools console shows the message written by C# code when the user selects the Yes! button in the UI.](~/blazor/index/_static/dialog.png)
-
-Components render into an in-memory representation of the browser's [Document Object Model (DOM)](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) called a *render tree*, which is used to update the UI in a flexible and efficient way.
-
-:::moniker range=">= aspnetcore-8.0"
-
-## Blazor Web Apps
+## Build a full-stack web app with Blazor
 
 Blazor Web Apps provide a component-based architecture with server-side rendering and full client-side interactivity in a single project, where you can switch between server-side and client-side rendering modes and even mix them in the same page. 
 
 When a Blazor Web App uses interactivity with server rendering, the server generates HTML in response to a request and sends it to the browser. The page loads fast because UI rendering is performed quickly on the server without the need to download a large JavaScript bundle or wait for the establishment of a [SignalR](xref:signalr/introduction) connection to the client.
```

Comment:
This sounds like it's talking about static server rendering, but it also mentions "interactivity". 

Also, we shouldn't assume that the reader understands anything about interactive server rendering at this point (websockets, SignalR, etc.). Interactive server rendering is a unique Blazor capability that deserves its own paragraph, something like: "Blazor also supports interactive server rendering, where UI interactions are handled from the server over a real-time connection with the browser. Interactive server rendering enables a rich user experience without the need to create API endpoints to access server resources."

```suggestion
Blazor Web Apps can deliver UI to the browser fast by statically rendering HTML content from the server in response to requests.  The page loads fast because UI rendering is performed quickly on the server without the need to download a large JavaScript bundle. Blazor can also further improve the user experience with various progressive enhancements to server rendering, like enhanced navigation & form posts and streaming rendering of asynchronously generated content.
```